src/common/colorspaces_inline_conversions.h

```diff
@@ -421,6 +422,125 @@ static inline float dt_camera_rgb_luminance(const float *const rgb)
   return (rgb[0] * 0.2225045f + rgb[1] * 0.7168786f + rgb[2] * 0.0606169f);
 }
 
+#ifdef _OPENMP
+#pragma omp declare simd
+#endif
+static inline void dt_XYZ_2_JzAzBz(const float *const XYZ_D65, float *const JzAzBz)
```

Comment:
could add `const float *const restrict` and `float *const restrict` to inform compiler there is no aliasing

Reply:
As the file is included by some C++ code there was some troubles to use restrict.

Reply:
right, I forgot.